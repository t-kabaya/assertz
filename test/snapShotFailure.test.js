const { snap } = require('../index')

snap('bar', 'string snap')
