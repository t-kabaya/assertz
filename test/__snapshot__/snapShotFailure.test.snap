[{"snap":"bar","testName":"string snap","path":"//Users/ipg/kbyProject/helloUnitTest/test/snapshotFailure.test.js"}]